mlflow/tracking/fluent.py

```diff
@@ -1107,6 +1116,12 @@ def search_runs(
         filter_string = "tags.s.release ILIKE '%rc%'"
         df = mlflow.search_runs([experiment_id], filter_string=filter_string)
         print(df[["metrics.m", "tags.s.release", "run_id"]])
+        print("--")
+
+        # Search for all the runs in the experiment with given experiment name
```

Comment:
```suggestion
        # Search for all of the runs in the experiment with the given experiment name
```

Reply:
Same as above, even Pycharm tells me that "of" looks suspicious, won't include it for now